src/main/resources/digital/slovensko/autogram/ui/gui/signatures-invalid-dialog.fxml

```diff
@@ -12,15 +12,15 @@
       prefWidth="768" minWidth="450"
       fx:id="mainBox">
 
-    <VBox styleClass="autogram-error-summary">
```

Comment:
Tuto pozor, ja som to robil 1:1 ku triedam govuk pokial sa dalo. akonahle sa vzdialime, tak to budeme musiet v hlave prekladat na nieco ine. https://design-system.service.gov.uk/components/error-summary/ zle sa s tym bude robit

Reply:
Rozumiem, ale nedá sa všetko spraviť 1:1. Napr. tieto obrazovky vymysleli UXáci, lebo sa to nedalo celé preklopiť z nejakého gov uk komponentu. Musíme mať aj custom veci.

Tie error-summary zostali, ibaže tieto obrazovky nie sú úplne error.

Reply:
Nevim teda, ale otvorim si ten design system govuk a vidim tam error summary a vyzera to uplne ako u nas. Preco to vlastne premenovavame? Ja chapem, ze u nas to je z biznis pohladu warning ale z pohladu komponentov idsk to je error (zatial). Nevidim velmi dovod sa vzdalovat od ich konvencie. Ak budeme mat vlastny komponent tak si ho nazvime ako chceme ale ich komponenty nazvime ako sa volaju.

Reply:
Ach sorry pravda.